test.rb

```diff
@@ -0,0 +1,36 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+def a
+  def b
+    1
+    ['a','c']
+    'C'
+    "A"
+    ?a
```

Comment:
__[development][rubocop]__
* Literal `'A'` used in void context.
* Do not use the character literal - use string literal instead. [:ref](https://github.com/bbatsov/ruby-style-guide#no-character-literals)

<!-- Comment by [SideCI](https://sideci.com) -->